clang/docs/MemorySanitizer.rst

```diff
@@ -8,11 +8,18 @@ MemorySanitizer
 Introduction
 ============
 
-MemorySanitizer is a detector of uninitialized reads. It consists of a
+MemorySanitizer is a detector of uninitialized memory use. It consists of a
 compiler instrumentation module and a run-time library.
 
 Typical slowdown introduced by MemorySanitizer is **3x**.
 
+Here is a not comprehensive list cases when MemorySanitizer will report an error:
+
+* A code uses uninitialized value in a conditional branch.
```

Comment:
Nit: use consistent tense (e.g., be consistent about "uses" vs. "was used" in each bullet point)